src/pages/SubmissionConfig.js

```diff
@@ -224,7 +224,7 @@ export const SubmissionConfig = ({ methodId }) => {
   useEffect(() => {
     async function getWorkflowScript() {
       try {
-        const workflowUrlRaw = reconstructToRawUrl(selectedMethodVersion.url)
+        const workflowUrlRaw = await convertToRawUrl(selectedMethodVersion.url, selectedMethodVersion.name, method.source)
```

Comment:
just curious... does this actually need `await`? It's not obvious to me why it would, but maybe I missed something

Reply:
I had to add await because we are making an Ajax call for Dockstore workflows